src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
+  const stylesObject = {};
+
+  stylesArray.forEach((style) => {
+    const [property, value] = style.split(':').map((item) => item.trim());
+
+    if (property && value) {
+      stylesObject[property] = value;
+    }
```

Comment:
The use of 'forEach' here is appropriate as it doesn't return a new array. However, ensure that the function handles potential edge cases such as properties without values or values without properties.